Sources/PackageGraph/PackageGraph.swift

```diff
@@ -74,7 +74,7 @@ public struct PackageGraph {
     ///
     //// This set will also have references to packages that are currently present
     /// in the graph due to loading errors. This set doesn't include the root packages.
-    public let requiredDependencies: Set<PackageReference>
+    public let requiredDependencies: [PackageReference]
```

Comment:
Should this be an ordered set instead?

Reply:
I think that at this stage an array is fine, it preservers the order of the original ordered set used to generate it, and there is not a functional need in a set downstream afaict

Reply:
OK, I don't have a strong opinion. Could we update the doc comment, though?